thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -318,8 +331,19 @@ public Response updateAnomalyFunction(@NotNull @QueryParam("id") Long id,
       String filterString = ThirdEyeUtils.getSortedFiltersFromJson(filters);
       anomalyFunctionSpec.setFilters(filterString);
     }
-    anomalyFunctionSpec.setExploreDimensions(exploreDimensions);
     anomalyFunctionSpec.setProperties(properties);
+    // Ensure that the explore dimension names are ordered as schema dimension names
```

Comment:
Previous one is for creation of anomaly function; this one is for update of anomaly function.